bp_top/test/tb/bp_tethered/Makefile.vcs

```diff
@@ -1,5 +1,7 @@
 VCS_TIMEOUT = 1000ms
 
+export ADDR_WIDTH ?= 40
```

Comment:
Handle this like shown here: https://github.com/black-parrot/black-parrot/blob/e7b13bf4f1836365d57e204c6a5344dcccd91cec/bp_top/test/tb/bp_tethered/Makefile.cfgs#L1

And just infer it for the non-40-bit configs

Reply:
Should I define a new cfg for 44 bit paddr?

Reply:
```
diff --git a/bp_top/test/tb/bp_tethered/Makefile.cfgs b/bp_top/test/tb/bp_tethered/Makefile.cfgs
index 297627bd..19841b8c 100644
--- a/bp_top/test/tb/bp_tethered/Makefile.cfgs
+++ b/bp_top/test/tb/bp_tethered/Makefile.cfgs
@@ -19,32 +19,22 @@ e_bp_multicore_3_cfg_cores              := 3
 e_bp_multicore_2_cce_ucode_cfg_cores    := 2
 e_bp_multicore_2_l2e_cfg_cores          := 4
 e_bp_multicore_2_cfg_cores              := 2
-e_bp_multicore_1_accelerator_cfg_cores  := 1
-e_bp_multicore_1_cce_ucode_bootrom_cfg_cores := 1
-e_bp_multicore_1_cce_ucode_cfg_cores    := 1
-e_bp_multicore_1_l1_medium_cfg_cores    := 1
-e_bp_multicore_1_l1_small_cfg_cores     := 1
-e_bp_multicore_1_no_l2_cfg_cores        := 1
-e_bp_multicore_1_bootrom_cfg_cores      := 1
-e_bp_multicore_1_cfg_cores              := 1
-e_bp_multicore_1_l2e_cfg_cores          := 2

-e_bp_unicore_writethrough_cfg_cores     := 1
-e_bp_unicore_l1_wide_cfg_cores          := 1
-e_bp_unicore_l1_hetero_cfg_cores        := 1
-e_bp_unicore_l1_tiny_cfg_cores          := 1
-e_bp_unicore_l1_medium_cfg_cores        := 1
-e_bp_unicore_l1_small_cfg_cores         := 1
-e_bp_unicore_no_l2_cfg_cores            := 1
-e_bp_unicore_bootrom_cfg_cores          := 1
-e_bp_unicore_cfg_cores                  := 1
-
-# Note, custom must be manually overidden here for cosim to work properly
-e_bp_custom_cfg_cores                   := 1
 e_bp_default_cfg_cores                  := 1

-export NCPUS ?= $($(CFG)_cores)
+_NCPUS :=$($(CFG)_cores)
+ifeq ($(_NCPUS),)
+  export NCPUS ?= 1
+else
+  export NCPUS ?= $(_NCPUS)
+endif
+
+e_bp_default_cfg_paddr_width                 := 40

-ifeq ($(origin $(CFG)_cores), undefined)
-$(error NCPUS not found for CFG=$(CFG))
+_PADDR_WIDTH :=$($(CFG)_paddr_width)
+ifeq ($(_PADDR_WIDTH),)
+  export PADDR_WIDTH ?= 40
+else
+  export PADDR_WIDTH ?= $(_PADDR_WIDTH)
 endif
```

Reply:
use this